source/documentation/tools/visual-studio-code/index.md

```diff
@@ -10,6 +10,8 @@
 
 * [Python 3.10](https://www.python.org/downloads/release/python-3100/)
 
+* [Ollama](/tools/visual-studio-code/ollama)
```

Comment:
Maybe link this this to Ollama's website or GitHub like the links above?

Reply:
The link is in the following section along with the instructions to serve and run ollama